src/main/java/org/cryptomator/ui/recoverykey/RecoveryKeyComponent.java

```diff
@@ -38,11 +43,19 @@ default void showRecoveryKeyRecoverWindow() {
 		stage.show();
 	}
 
+	default void showIsHubVaultDialogWindow() {
+		Stage stage = window();
+		stage.setScene(recoverIsHubVaultScene().get());
+		stage.sizeToScene();
+		stage.show();
+	}
 
 	@Subcomponent.Factory
 	interface Factory {
 
-		RecoveryKeyComponent create(@BindsInstance @RecoveryKeyWindow Vault vault, @BindsInstance @Named("keyRecoveryOwner") Stage owner);
+		RecoveryKeyComponent create(@BindsInstance @RecoveryKeyWindow Vault vault, //
+									@BindsInstance @Named("keyRecoveryOwner") Stage owner, //
+									@BindsInstance @Named("recoverType") ObjectProperty<RecoverUtil.Type> recoverType);
```

Comment:
is this property ever changed/inteded to be changable? If not, skip the property wrapper and use the RecoverUtil.Type directly.